engine/flink/management/src/it/scala/pl/touk/nussknacker/engine/management/streaming/ClassPaths.scala

```diff
@@ -7,7 +7,9 @@ object ClassPaths {
   val commonClasspath = List(
     s"./engine/flink/components/base/target/scala-${ScalaMajorVersionConfig.scalaMajorVersion}/flinkBase.jar",
     s"./engine/flink/components/kafka/target/scala-${ScalaMajorVersionConfig.scalaMajorVersion}/flinkKafka.jar",
-    s"./engine/flink/executor/target/scala-${ScalaMajorVersionConfig.scalaMajorVersion}/flinkExecutor.jar"
+    s"./engine/flink/executor/target/scala-${ScalaMajorVersionConfig.scalaMajorVersion}/flinkExecutor.jar",
+    "./engine/flink/executor/it-libs/flink-additional-deps/flink-metrics-dropwizard.jar",
```

Comment:
Shouldn't we put these libs into some target? Doesn't git try to track them?

Reply:
ah, I forgot to commit .gitignore file. Do you think it would be better to place them under target directory?

Reply:
They are kind of products of our build script so IMO it is fine to keep them in the `target` directory. Maybe someone could want to invalidate them and wanted to do it via `sbt clean`? Also, it is nice to have as few directories as possible in modules directories.